packages/fluentui/react-northstar/src/themes/teams/components/Popup/popupContentStyles.ts

```diff
@@ -47,5 +53,9 @@ export const popupContentStyles: ComponentSlotStylesPrepared<PopupContentStylesP
     ...(p.pointing && {
       pointerEvents: 'all',
     }),
+
+    ...(p.autoSize && {
+      overflow: 'auto',
+    }),
```

Comment:
```suggestion
    ...(p.autoSize && {
      overflow: 'auto',
    }),
    ...(p.autoSize === 'height' && {
    }),
    ...(p.autoSize === 'width' && {
    }),
```

Should we have separate conditions for `autoSize="height"` and `autoSize="width"` to apply overflow for a proper axis?